tensorflow_io/json/README.md

```diff
@@ -0,0 +1,207 @@
+# TensorFlow i/o JSON Datasets
+
+[JSON (JavaScript Object Notation)](https://www.json.org/) is an open-standard
+file format that uses human-readable text to transmit data objects consisting
+of key–value pairs and array data types (or any other serializable value). JSON
+format is widely used in machine learning and data science.
+
+TensorFlow i/o JSON support mainly focuses on the **record** array in JSON, here
```

Comment:
It's a little vague what you mean by "mainly focuses". Are any other formats supported or does it need to be in record arrays?

Reply:
Yeah, that's right, currently, it only supports the record arrays. I will update the PR